manifests/addons/vault.yml

```diff
@@ -0,0 +1,23 @@
+---
+params:
+  vault_url: '(( param "Please specify the URL of the Vault for Concourse to pull
+    credentials from" ))'
+  vault_path_prefix: "/concourse"
+  vault_insecure_skip_verify: false
+  vault_token: '(( param "Please specify the token that Concourse should use to
+    authenticate with Vault" ))'
+
+
+
+instance_groups:
+- name: web
+  jobs:
+  - name: atc
+    properties:
+      vault:
+        url: (( grab params.vault_url ))
+        path_prefix: (( grab params.vault_path_prefix ))
+        tls:
+          insecure_skip_verify: (( grab params.vault_insecure_skip_verify ))
```

Comment:
Rather than defining defaults up top, use Spruce's `||` notation:

```
(( grab params.vault_insecure_skip_verify || false ))
```

It leads to a cleaner manifest.